config.default.yaml

```diff
@@ -691,6 +703,7 @@ sector:
 
   h2_cavern: true
   marginal_cost_storage: 0
+  fischer_tropsch: true
```

Comment:
Would be great to group the enabling parameter for Fischer-Tropsch with another parameter available for same technology to facilitate navigation. I mean this one: https://github.com/pypsa-meets-earth/pypsa-earth/blob/f18a57c3557910f90a902ccca113e1f5bcf532db/config.default.yaml#L702